internal/system/nvdevices/devices_test.go

```diff
@@ -46,6 +60,7 @@ func TestCreateControlDevices(t *testing.T) {
 		root          string
 		devices       devices.Devices
 		mknodeError   error
+		hasError      bool
```

Comment:
Ideally `require.ErrorIs` should be used. If we're wrapping errors correctly, then things should work as expected.